sema/type.go

```diff
@@ -9511,10 +9541,49 @@ func (t *EntitlementMapType) CheckInstantiated(_ ast.HasPosition, _ common.Memor
 	// NO-OP
 }
 
+func extractNativeTypes(
+	types []Type,
+) {
```

Comment:
Need to return the resulting `slice` after all the appends, because the `append` function allocates a new slice (unlike maps). So any appends made inside this function would be lost.

Reply:
ah right!